File: Patterns/python/pattern2.py
# n = 4
# 1111
# 2222
# 3333
# 4444

n = int(input())
row = 1
while row <= n:
    col = 1
    while col <= n:
        print(row, end="");
        col+=1
    row += 1
    print("")

File: Patterns/go/pattern2.go
// n = 4

// 1
// 2 3
// 4 5 6
// 7 8 9 10

package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	row := 1
	val := 1
	for row <= n {
		col := 1
		for col <= row {
			fmt.Printf("%d", val)
			col++
			val++
		}
		row++
		fmt.Printf("\n")
	}
}